### PR TITLE
Adding independent checks for both drive letter and volume labels.

### PR DESCRIPTION
**Pull request by brynsp**
This change is [Reviewable](https://reviewable.io/reviews/powershell/xstorage/44)

### CODE REVIEW

File: DSCResources/MSFT_xDisk/MSFT_xDisk.psm1 [hunk 1]
```diff
@@ -143,9 +143,20 @@ function Set-TargetResource
         }
         else 
         {
-            Write-Verbose -Message "The volume already exists, adjusting drive letter..."
-            $VolumeDriveLetter = ($Disk | Get-Partition | Get-Volume).driveletter
-            Set-Partition -DriveLetter $VolumeDriveLetter -NewDriveLetter $DriveLetter
+            $Volume = $Disk | Get-Partition | Get-Volume
```
Comment (TravisEz13) [inline DSCResources/MSFT_xDisk/MSFT_xDisk.psm1]: Please add or update  test coverage for the scenario you added code for.

It is required that you provide adequate test coverage for the code you change. [Contribution Guidelines -Adding test coverage for DSC resources.](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#adding-test-coverage-for-dsc-resources)  Please work with us to come up with a plan to cover this code.

**Comment by TravisEz13** [comment]
Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)

**Comment by TravisEz13** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xstorage/44)*

**Comment by gigi81** [comment]
This is a duplicate of the changes in PR #38

**Comment by TravisEz13** [comment]
Please resolve all merge conflicts.  For more information see [GitHowTo - 30. Resolving Conflicts](http://githowto.com/resolving_conflicts )
[Instructions to setup BC4, my favorite merge tool](http://www.scootersoftware.com/support.php?zz=kb_vcs#gitwindows)

**Comment by TravisEz13** [comment]
If there is no update of the pull request within 7 days, we will close the pull request.  After that, if you have an update, feel free to reopen the pull request or a new pull request.  Please consider closing and re-opening when ready for review.

**Comment by PlagueHO** [comment]
@TravisEz13 - Can we please close this PR as I've completed #53 now (submitting the PR tonight) and it basically reworks this resource completely for HQRM and many other fixes and features (it includes the fix or PR #38). Besides, as @gigi81 says this is actually a duplicate fix.

**Pull request closed without merging**